Drivers/Src/MotorController.c

```diff
@@ -0,0 +1,68 @@
+#include "MotorController.h"
+
+
+#define MOTOR_DRIVE 0x221
+#define VELOCITY_ID 0x243
+#define SUCCESS     1
+#define FAILURE     0
```

Comment:
These seem pretty generic. Are there global error codes available outside of the MotorController.c file that could be used instead (something common to all drivers/apps)?

Reply:
Nothing yet, but we should add something in `config.h` that can be used globally. I'll make sure that happens.